flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/stream/MultiJoinTestPrograms.java

```diff
@@ -1144,4 +1144,98 @@ public class MultiJoinTestPrograms {
                                     + "FROM UsersNullSafe u "
                                     + "INNER JOIN OrdersNullSafe o ON u.user_id IS NOT DISTINCT FROM o.user_id")
                     .build();
+
+    public static final TableTestProgram MULTI_JOIN_CONDITIONS_MATERIALIZATION_NEXMARK_AGGREGATION =
+            TableTestProgram.of(
+                            "nexmark-auction-bid-aggregation",
```

Comment:
nit: I see you have named the constant Nexmark, I think it would be better to include the test description/goal in the constant name , similar to MULTI_JOIN_NULL_SAFE_JOIN_WITH_NULL_KEYS.